paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/ui/wallet/SavedPaymentMethodsViewModel.kt

```diff
@@ -28,51 +30,75 @@ class SavedPaymentMethodsViewModel(
     )
     val state: StateFlow<SavedPaymentMethodsViewState> = _state
 
-    init {
-        initialize()
-    }
+    internal val customerAdapter: CustomerAdapter = CustomerAdapter.create(
+        context = getApplication(),
+        customerEphemeralKeyProvider = {
+            fetchCustomerEphemeralKey().fold(
+                success = {
+                    kotlin.Result.success(
+                        CustomerEphemeralKey.create(
+                            customerId = it.customerId,
+                            ephemeralKey = it.customerEphemeralKeySecret
+                        )
+                    )
+                },
+                failure = {
+                    kotlin.Result.failure(it.exception)
+                }
+            )
+        },
+        setupIntentClientSecretProvider = null,
+        canCreateSetupIntents = false
```

Comment:
I don't really know what this is doing right now, but it _feels_ like we might be able to compute this? If `setupIntentClientSecretProvider` is null, can we just assume this is false? (Jay is being very naive here).

Reply:
Yea, this should be computed. Good catch!